lib/UmdTemplatePlugin.js

```diff
@@ -146,7 +146,9 @@ class UmdTemplatePlugin {
 								.map(m => {
 									let request = m.request;
 									if (typeof request === "object") request = request.root;
-									return `root${accessorToObjectAccess([].concat(request))}`;
+									return `root${accessorToObjectAccess(
```

Comment:
If you have some time left you may replace `accessorToObjectAccess` with `lib/util/propertyAccess` in all files you find it (I think other library plugins also use it). That would help to unify this logic and makes library output a little more pretty. e. g. `root["alpha"]["beta"]` -> `root.alpha.beta`

Reply:
good point, will implement it soon